sdk/scheduler/src/test/java/com/mesosphere/sdk/http/PodResourceTest.java

```diff
@@ -116,6 +119,16 @@
 
     private PodResource resource;
 
+    @BeforeClass
+    public static void beforeAll() throws InterruptedException {
```

Comment:
Something something, Singletons are the root of all evil

Reply:
False.